### ISSUE TITLE
Improve test suite

### ISSUE THREAD

**Issue by ben-eb** [frontendfriends/node-combine-mq#19]
Current tests are far too manual; I should be able to change the source code, and test and see if it broke. I think this is partly due to the fact that your module bundles disk operations in its main core, rather than its CLI wrapper where it should be.

**Comment by furzeface** [comment]
Yes, it is something I'd like to improve when we have time. Thanks for the advice 😀

**Comment by furzeface** [comment]
Link Issue #15
